utils/dicom_send.py

```diff
@@ -103,7 +103,7 @@ def exit_if_archive_empty(archive_obj):
 
 
 def download_and_send(
-    api_key,
+    fw,
```

Comment:
This really doesn't need to be passed as an argument. Why not just import flywheel again, like you would any other library, pass session_id from config via parse_config.py?